### PR TITLE
fix(ci/cd): sync with variable secrets

### PR DESCRIPTION
**Pull request by ayocodingit**
## Overview

- fix(ci/cd): sync with variable secrets

## Evidence
- title: fix(ci/cd): sync with variable secrets
- project: Portal Jabar
- participants: @ayocodingit @feriahmad

### CODE REVIEW

File: .github/workflows/app-build.yml [hunk 1]
```diff
@@ -1,60 +1,61 @@
...
+  push:
+    branches: [main]
 
 jobs:
-    deploy:
-        name: Setup, Build, Publish, and Deploy
-        runs-on: ubuntu-latest
-        steps:
-            - name: Checkout
-              uses: actions/checkout@v2
```
Comment (feriahmad) [inline .github/workflows/app-build.yml:13]: uses: actions/checkout@v3

File: .github/workflows/app-build.yml [hunk 2]
```diff
@@ -1,60 +1,61 @@
...
-
-            - name: Set up QEMU
-              uses: docker/setup-qemu-action@v1
```
Comment (feriahmad) [inline .github/workflows/app-build.yml:16]: uses: docker/setup-qemu-action@v2

File: .github/workflows/app-build.yml [hunk 3]
```diff
@@ -1,60 +1,61 @@
...
-
-            - name: Set up Docker Buildx
-              uses: docker/setup-buildx-action@v1
```
Comment (feriahmad) [inline .github/workflows/app-build.yml:19]: uses: docker/setup-buildx-action@v2

File: .github/workflows/app-build.yml [hunk 4]
```diff
@@ -1,60 +1,61 @@
...
-              run: |
-                  docker login ${{ secrets.REGISTRY_URL }} -u ${{ secrets.REGISTRY_USERNAME_MICROSITE }} -p "${{ secrets.REGISTRY_PASSWORD_MICROSITE }}"
-              # Build the Docker image
-            - name: Set version tag
-              id: vars
-              run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
-
-            - name: Build and push
-              id: docker_build
-              uses: docker/build-push-action@v2
```
Comment (feriahmad) [inline .github/workflows/app-build.yml:32]: uses: docker/build-push-action@v4

**Comment by ayocodingit** [comment]
done, your are comment @feriahmad

File: .github/workflows/app-build.yml [hunk 5]
```diff
@@ -1,60 +1,61 @@
...
+
+      # Configure Docker with Credentials
+      - name: Configure Docker
+        run: |
+          docker login ${{ secrets.REGISTRY_URL }} -u ${{ secrets.REGISTRY_USERNAME }} -p "${{ secrets.REGISTRY_PASSWORD }}"
+
+      # Build the Docker image
+      - name: Set version tag
+        id: vars
+        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
```
Comment (feriahmad) [inline .github/workflows/app-build.yml:29]: change to run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_ENV

**Review by feriahmad — APPROVED** [review]

**Pull request merged**